easybuild/easyconfigs/h/HOMER/HOMER-4.7-goolf-1.7.20.eb

```diff
@@ -0,0 +1,41 @@
+easyblock = 'MakeCp'
+
+name = 'HOMER'
+version = '4.7'
+
+homepage = 'http://homer.salk.edu/homer/'
+description = """HOMER (Hypergeometric Optimization of Motif EnRichment) is a suite of tools for Motif Discovery and 
+ next-gen sequencing analysis.  It is a collection of command line programs for unix-style operating systems written in
+ Perl and C++. HOMER was primarily written as a de novo motif discovery algorithm and is well suited for finding 8-20 
+ bp motifs in large scale genomics data.  HOMER contains many useful tools for analyzing ChIP-Seq, GRO-Seq, RNA-Seq, 
+ DNase-Seq, Hi-C and numerous other types of functional genomics sequencing data sets."""
+
+toolchain = {'name': 'goolf', 'version': '1.7.20'}
+
+source_urls = ['http://homer.salk.edu/homer/data/software/']
+sources = ['homerCppOnly.v%(version)s.zip']
+
+
+executables = ['makeTagDirectory',
+     'findPeaks',
+     'mergePeaks',
+     'genomeOntology',
+     'getPeakTags',
+     'makeUCSCfile',
+     'getDifferentialPeaks',
+     'assignGenomeAnnotation',
+     'homerTools',
+     'homer',
+     'homer2',
+     'analyzeHiC',
+     'getGenomeTilingPeaks'
+ ]
+files_to_copy = [(executables, 'bin')]
```

Comment:
This seems to suggest that this easyconfig should be using `ConfigureMake` as an easyblock, and skip the install step + precreate the `bin` subdir via `prebuildopts`?

```python
easyblock = 'ConfigureMake'
...
skipsteps = ['install']
prebuildopts = "mkdir -p %(installdir)s/bin && "
```

Would that work?